static/app/views/onboarding/sdkConfiguration.tsx

```diff
@@ -16,8 +17,9 @@ const SdkConfiguration = (props: Props) => {
   // check for manual override query param
   if (integrationSlug && parsed.manual !== '1') {
     return <IntegrationSetup integrationSlug={integrationSlug} {...props} />;
-  }
-  return <ProjectSetup {...props} />;
+  } else if (platform === 'other') {
+    return <OtherSetup {...props} />;
+  } else return <DocumentationSetup {...props} />;
```

Comment:
@roggenkemper Nit: we don't need the `else` statement here

Reply:
removed!